src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/StreamInvocationMessage.java

```diff
@@ -3,16 +3,26 @@
 
 package com.microsoft.signalr;
 
+import java.util.Collection;
+
 final class StreamInvocationMessage extends HubMessage {
```

Comment:
Why does this extend HubMessage instead of InvocationMessage?

Reply:
If I remember correctly it had to do with how the messages get serialized. I remember making the same comment a while back when we first introduced this message type. I'll double check

Reply:
Updated `StreamInvocationMessage` to extend from `InvocationMessage`.